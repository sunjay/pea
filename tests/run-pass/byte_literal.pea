fn main() {
    let a = b'a';
    let b = b'b';
    let n = b'\n';

    println!(a);
    println!(b);
    println!(n);
    println!(b);
    println!(a);

    foo(a);
    foo(b);
}

fn foo(x: u8) {
    print!(b"check out this byte: ");
    println!(x);
}
