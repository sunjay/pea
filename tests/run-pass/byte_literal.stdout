97
98
10
98
97
check out this byte: 97
check out this byte: 98
